tasks/assembly/task_raven.wdl

```
@@ -0,0 +1,55 @@
+version 1.0
+
+task raven {
+  input {
+    File read1
+    String samplename
+
+    Int raven_polishing_iterations = 2
+    Float? raven_identity = 0
+    String? raven_opts # Any extra Raven-specific parameters
+
+    Int cpu = 4
+    Int disk_size = 100
+    String docker = "us-docker.pkg.dev/general-theiagen/theiagen/raven:1.8.3"
+    Int memory = 16
+  }
+  command <<<
+    # fail hard
```

no you don't

---

why did you resolve this you don't actually fail hard